roop/core.py

```diff
@@ -209,6 +209,8 @@ def start(preview_callback = None):
         process_video_multi_cores(args.source_img, args.frame_paths)
     else:
         process_video(args.source_img, args.frame_paths)
+    if args.gpu_vendor = 'nvidia':
```

Comment:
should be `args.gpu_vendor == 'nvidia':`

Reply:
sorry typing too quick